packages/devtools_app/lib/src/service/service_manager.dart

```diff
@@ -136,18 +136,17 @@ class ServiceConnectionManager {
   /// Call a service that is registered by exactly one client.
   Future<Response> callService(
     String name, {
-    String isolateId,
-    Map args,
+    String? isolateId,
+    Map? args,
```

Comment:
add `<String, dynamic>` here and get rid of the type cast below

Reply:
done